### DISCUSSION TITLE
How to relink to Plex server?

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [Taxel/PlexTraktSync#1960] [Q&A] [answered]
In short, I woke up yesterday to my Plex server randomly being offline. After lots of work I was able to get it fixed again, but it lost my claim token, so I had to reclaim my server. It's all back up and running, but the Plex token has been changed. When I try to run sync, I'm met with this following error:

```
INFO     PlexTraktSync [0.30.9]                                                                         
INFO     Connecting with url: http://###.###.###.###:32400, timeout 30 seconds                            
ERROR    (401) unauthorized; http://###.###.###.###:32400/ <html><head><script>window.location =          
         window.location.href.match(/(^.+\/)[^\/]*$/)[1] +                                              
         'web/index.html';</script><title>Unauthorized</title></head><body><h1>401                      
         Unauthorized</h1></body></html>                                                                
CRITICAL Error running sync command: No more methods to connect. Giving up  
```

I'm assuming this is because of the token being reset. I know how to get the general Plex token from a library item's Get Info XML in Plex, but how do I get account token for users in my Plex Home who have their own Plex accounts? Do I have to sign in as them with their account to get it the same way? These are not standard managed accounts, but separate user accounts that I added to my Plex Home so they have their own logins/accounts.

**Comment by SawkeeReemo** [comment]
UPDATE: [I found this comment on Reddit](https://www.reddit.com/r/PleX/comments/18ihi91/comment/kddct4k/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button), which I will copy/paste below. Going to use that to attempt to grab the token.

> Only the admin has the 'View XML' option, so shared users would have to use a different method for finding their auth token (e.g. using browser dev tools and look for X-Plex-Token in requests made to your server).
> 
> You can also find shared users auth tokens that are scoped to your server. One manual way of doing this is to:
> 
> Go to https://plex.tv/api/resources?X-Plex-Token={your_admin_token} and find the clientIdentifier of your server.
> 
> Go to [https://plex.tv/api/servers/{clientIdentifier}/shared_servers?X-Plex-Token={your_admin_token}](https://plex.tv/api/servers/%7BclientIdentifier%7D/shared_servers?X-Plex-Token={your_admin_token}), where all your shared users will be listed, along with their accessToken for the server.

**Comment by SawkeeReemo** [comment, marked as answer]
Another UPDATE for anyone searching --

Instead of manually adding the tokens in, running either:
`docker compose run --rm plextraktsync login`
or
`docker compose run --rm plextraktsync plex-login`

fixes the connection issue for me. Hopefully this helps anyone else who may be searching for a solution to this problem.

**Reply by glensc** [reply]
NOTE `login` will only relogin if token is missing, so it token is present but invalid, it won't help you. you need to use `plex-login` to force re-login.

**Reply by SawkeeReemo** [reply]
Thanks for the detailed response on those. I think for me, I had one instance of plextrakt sync that lost its token all together which is why I ran `login` first. (My fault, not a bug or anything.) But regardless, this did a great job of reconnecting and everything has been working great. Appreciate your hard work on this!
